File: fixtures/hub-cobol-clbs-mini/batch/reference_emit_rptaurn.py
"""Reference emit for RPTAUDRN — CLBS RPTAUD00-shaped audit+error severity sum → 85.75."""
# EXPECTED: 85.75
AUDIT = [20.00, 35.50, 15.25]
ERRORS = [10.00, 5.00]
print(f"{sum(AUDIT) + sum(ERRORS):.2f}")

File: fixtures/hub-cobol-clbs-mini/batch/reference_emit_rptaurn.cs
// EXPECTED: 85.75
// Reference emit for RPTAUDRN — CLBS audit-report sequential severity sum.
using System;
public static class RptaurnRef {
  public static void Main() {
    double[] audit = {20.00, 35.50, 15.25};
    double[] errors = {10.00, 5.00};
    double total = 0.0;
    foreach (var v in audit) total += v;
    foreach (var v in errors) total += v;
    Console.WriteLine(total.ToString("0.00"));
  }
}
